app/models/mappers/check-activities.js

```diff
@@ -11,18 +169,211 @@ const getActivityLabelFromEvent = (event) => {
   }
   return 'NOT YET DEFINED'
 }
+/**
+ * @typedef ActivityRow
+ * @property {string} activityLabel
+ * @property {string} date
+ * @property {string} teamMember
+ */
 
-const mapActivityDtoToCheckActivityRow = (event) => {
+/**
+ * @param {DDIEvent} event
+ * @returns {Omit<ActivityRow, 'activityLabel'>}
```

Comment:
This return comment doesn't seem correct

Reply:
it's a TypeScript type (seems to work), Omits activityLabel prop from ActivityRow